_posts/2019-03-19-gsod2019.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: Google Season of Docs 2019 Application
```

Comment:
The date of the post needs to be changed – perhaps just set it to the application deadline, *2019-04-23-gsod2019.md*.

(Btw, the date was WRONG... it's April, not March, so it needs to be fixed in any case!!)